src/rantanplan/core.py

```diff
@@ -38,9 +54,1266 @@
 LIAISON_FIRST_PART = set("aeiouáéíóúAEIOUÁÉÍÓÚy")
 LIAISON_SECOND_PART = set("aeiouáéíóúAEIOUÁÉÍÓÚhy")
 STRESSED_UNACCENTED_MONOSYLLABLES = {"yo", "vio", "dio", "fe", "sol", "ti", "un"}
-UNSTRESSED_UNACCENTED_MONOSYLLABLES = {"me", "nos", "te", "os", "lo", "la", "los", "las", "le",
-                                       "les", "se", "tan", "el", "mas", "te", "si", "tu", "de",
-                                       "mi", "si", "tu", "que", "de", "su"}
+UNSTRESSED_UNACCENTED_MONOSYLLABLES = {'de', 'el', 'la', 'las', 'le', 'les', 'lo', 'los',
+                                       'mas', 'me', 'mi', 'nos', 'os', 'que', 'se', 'si',
+                                       'su', 'tan', 'te', 'tu'}
+
+
+"""
+Regular expressions and rules for syllabification exceptions
+
+"""
+
+
+"""
+Fonetic exceptions dict for potential diaeresis
+
+paranoia': (['pa', 'ra', 'noia'],
+         [
+         (['pa', 'ra', 'no', 'ia'], (2, 3)), # Metaplasm positions
```

Comment:
Shouldn't this be more like "Position where further metaplasms should not be applied (since they are already applied)"?

Reply:
It's the position to mark as metaplasms if they are used

Reply:
Gotcha